game.js

```diff
@@ -19,13 +20,27 @@ function initGame() {
 
     reset();
 }
+function displayFuel(fuel){
+
+    document.getElementById('fuel').innerHTML ='Fuel is : '+fuel;
+    }	
 
 function loop() {
   
     if (thrust) {
+        if(fuel>0){
+            fuel-=1;
+            displayFuel(fuel);
+
+                    
+        }
+        if(fuel==0){
```

Comment:
This isn't right, when the fuel goes to 0 that just means you can't make the ship fly anymore, eventually it'll drop and crash.  Crashing here is premature

Reply:
ok I'll do it , and thanks for confirming..

Reply:
![fuelguage](https://user-images.githubusercontent.com/40556130/41928594-5816c4d0-7993-11e8-939d-90e40e29fda6.gif)